sanic/app.py

```diff
@@ -737,15 +738,14 @@ async def handle_request(self, request: Request):
             request.route = route
 
             if (
-                request.stream.request_body  # type: ignore
+                request.stream
+                and request.stream.request_body
                 and not route.ctx.ignore_body
             ):
```

Comment:
This change seems unrelated, accidentally included in this PR?

Reply:
`mypy` was complaining. We could use a type ignore, but I think this should be a light enough change.